src/Microsoft.FeatureManagement/Allocation/PercentileAllocation.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// The definition of a percentile allocation.
+    /// </summary>
+    public class PercentileAllocation
+    {
+        /// <summary>
+        /// The name of the variant.
+        /// </summary>
+        public string Variant { get; set; }
+
+        /// <summary>
+        /// The lower bound of the percentage to which the variant will be assigned.
+        /// </summary>
+        public double From { get; set; }
+
+        /// <summary>
+        /// The upper bound of the percentage to which the variant will be assigned.
```

Comment:
```suggestion
        /// The exclusive upper bound of the percentage to which the variant will be assigned.
```

Reply:
I want to make it clear that they can allocate 0-50 and 50-75, using 50 twice.

However this has minor impact, with 100 being an exception, so leave it out if you prefer.

Reply:
Makes sense to include it, just helps clarify how it works.